ipinfo/handler.py

```diff
@@ -123,9 +123,15 @@ def getDetails(self, ip_address=None, timeout=None):
         if response.status_code == 429:
             raise RequestQuotaExceededError()
         if response.status_code >= 400:
-            error_response = response.json()
             error_code = response.status_code
+
+            content_type = response.headers.get('content-type')
```

Comment:
Two final formatting/linting nitpicks:
- remove blank lines 127 and 134 (and lines 149 and 156 in `handler_async.py`)
- even though HTTP headers are case-insensitive, we prefer to use the capitalized form in our code

Reply:
For the header, I was trying to follow the existing pattern on those files. I only saw examples with lower case there.

Reply:
Sorry for the confusion, @PriOliveira  🤦 !  I'll update the older lowercase convention in our code in a follow-up PR.